src/Files.App/UserControls/AddressToolbar.xaml.cs

```diff
@@ -15,8 +15,17 @@ namespace Files.App.UserControls
 	public sealed partial class AddressToolbar : UserControl
 	{
 		private readonly IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
+		private readonly MainPageViewModel MainPageViewModel = Ioc.Default.GetRequiredService<MainPageViewModel>();
```

Comment:
Accessing `MainPageViewModel` from AddressToolbar isn't the best practice, but it'll do for now given the state of the rest of the control.